docs/requirements.txt

```diff
@@ -2,4 +2,5 @@ ipython
 scipy
 numpy > 1.9.0
 scikit-learn >= 0.18
+pandas
```

Comment:
@kmike I still don't understand the failure here: https://travis-ci.org/TeamHG-Memex/eli5/jobs/239548322#L508 - it tries to import ``pandas`` and fails, and it's indeed imported at the module level, but it's the same for lightdb/xgboost/lightning, so I don't understand yet why they don't fail the docs build, but pandas does fail.
On the other hand, don't we want to check all libraries docs when doing the travis docs build? So maybe it makes sense to include all optional ones here?